plugins/tedge_log_plugin/src/main.rs

```diff
@@ -43,11 +44,11 @@ pub struct LogfilePluginOpt {
     #[clap(long = "config-dir", default_value = DEFAULT_TEDGE_CONFIG_PATH)]
     pub config_dir: PathBuf,
 
-    #[clap(long, default_value = "te")]
-    root: String,
+    #[clap(long)]
+    mqtt_topic_root: Option<Arc<str>>,
```

Comment:
Renamed flags here to be consistent with `tedge-agent`, also because I think `--root` or `--device` do not give much context to the user of what these options are for, although probably this should be accompanied by help text.

Reply:
Understood. Consistency is most important!